### PR TITLE
Update Examples to stop using deprecated versions

### LINKED ISSUE #967: Examples use deprecated API since k8s v1.16
**Link to the issue (please include a link to the specific documentation or example)**:
This is the non-critical part of #963 

**Description of the issue (please include outputs or screenshots if possible)**:
In k8s v1.16, a few old APIs got deprecated. See #966 and kubernetes/kubernetes#70672. Also see https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/
Need update:
- [ ] In examples/notebooks, the notebook ```create_deployment.ipynb``` uses apps/v1beta1 API.
- [ ] In examples/notebooks, the notebook ```intro_notebook.ipynb``` uses apps/v1beta1 API.

Could use an update:
- [ ] The example ```ingress_create.py``` uses networking/v1beta1 API. This could be upgraded to networking/v1 API.

/assign

### PR DESCRIPTION
**Pull request by MoShitrit**
Fixes #967

### CODE REVIEW

**Comment by MoShitrit** [comment]
Hey @roycaihw!
I was wondering if you had a chance to take a look at the travisCI webhook issue yet?
I'd love to get this merged if everything looks good 😄 
Thanks!

File: examples/notebooks/create_deployment.ipynb [hunk 1]
```diff
@@ -2,345 +2,283 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "metadata": {
-    "deletable": true,
-    "editable": true
-   },
```
Comment (roycaihw) [inline examples/notebooks/create_deployment.ipynb:8]: Are these changes auto-generated? If so, could you split the generated change into a separate commit? That will make it easier to review

Comment (MoShitrit) [reply examples/notebooks/create_deployment.ipynb:8]: I actually edited these files in an IDE that has an integration with Jupyter notebook integration (IntelliJ IDEA, fwiw) so it gives a pretty convenient way to edit the file in a convenient way.
Nothing was auto-generated here.
I guess it might've added some additional fields which it wasn't supposed to.
Let me see if I can reduce the amount of changes here.

Comment (MoShitrit) [reply examples/notebooks/create_deployment.ipynb]: @roycaihw OK I was able to clean up the mess that was made there, it's looking better now.
To sum up: I updated the versions to use the current ones, and removed the examples that are no longer relevant (`Create DeploymentRollback object`). There was also an empty cell at the bottom that didn't seem to have any purpose, so I took the liberty of removing that as well.
Please let me know if any further changes are needed.
Thanks!

**Comment by palnabarun** [comment]
Thanks @MoShitrit for updating the examples.

/lgtm
